components/x-follow-button/src/FollowButton.jsx

```diff
@@ -12,6 +12,10 @@ const followButtonActions = withActions((props) => ({
 		console.log('the follow button was pressed');
```

Comment:
fyi this is expected behaviour as of ab50b0486c1d1a677e3b74733e21b356d09e2807. in reality the props aren't going to update in the way that knobs change them

Reply:
👍